src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -22,6 +22,7 @@
 using Stratis.Bitcoin.Utilities;
 
 [assembly: InternalsVisibleTo("Stratis.Bitcoin.Features.Consensus.Tests")]
+[assembly: InternalsVisibleTo("Stratis.Bitcoin.IntegrationTests")]
```

Comment:
It feels wrong to have to expose internals to the tests (I'm not saying it necessarily is, it just feels like it is getting a cheaty hook into the code rather than coming in at it from the public method angle).

Reply:
Well not necessarily the whole point of this feature is for testing if I am not mistaken.